### PR TITLE
Always define some Grakn env vars.

### PR DESCRIPTION
**Pull request by burukuru**

### CODE REVIEW

File: grakn-dist/src/grakn [hunk 1]
```diff
@@ -40,11 +40,12 @@ WAIT_INTERVAL_S=2
 if [ -z "${GRAKN_HOME}" ]; then
     [[ $(readlink $0) ]] && path=$(readlink $0) || path=$0
     GRAKN_HOME=$(cd "$(dirname "${path}")" && pwd -P)
-    GRAKN_CONFIG="${GRAKN_HOME}/conf/grakn.properties"
-    GRAKN_PID=/tmp/grakn.pid
-    GRAKN_STARTUP_TIMEOUT_S=120
 fi
 
+GRAKN_CONFIG="${GRAKN_HOME}/conf/grakn.properties"
```
Comment (aelred) [inline grakn-dist/src/grakn]: This one should still stay in the `if` because it refers to `${GRAKN_HOME}`

Comment (burukuru) [reply grakn-dist/src/grakn]: The if clause sets GRAKN_HOME when it's empty.
If it's pre-defined, we still want to set GRAKN_CONFIG.

Comment (aelred) [reply grakn-dist/src/grakn]: oh yeah! thanks for the wisdom

Comment (burukuru) [reply grakn-dist/src/grakn]: Thanks for the merge!

**Pull request merged**